app/controllers/eoy_donations_controller.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+class EoyDonationsController < ApplicationController
+  before_action :set_member
+
+  def opt_out
+    @eoy_donation_email = EoyDonationEmail.new(@member)
+
+    if @eoy_donation_email.opt_out
+      flash.now[:notice] = 'You have successfully opted out EOY donation email'
```

Comment:
one final request, can we please get these strings internationalised.

Reply:
yes. I will add that

Reply:
@osahyoun 
I have completed the changes. Shall you review this and merge?